data-protocols/ids/ids-api-multipart-endpoint-v1/src/main/java/org/eclipse/dataspaceconnector/ids/api/multipart/IdsMultipartApiServiceExtension.java

```diff
@@ -186,4 +194,7 @@ private String resolveConnectorId(@NotNull ServiceExtensionContext context) {
         return value;
     }
 
+    private Boolean resolveValidateReferring(@NotNull ServiceExtensionContext context) {
+        return Boolean.parseBoolean(context.getSetting(EDC_IDS_VALIDATION_REFERRINGCONNECTOR, null));
```

Comment:
A default is needed, otherwise it will throw exception on `TokenValidation` instantiation. Could be nice to introduce a boolean default method on `SettingResolver` and consequently on `Config` to have the possibility to retrieve a boolean setting

Reply:
Isn't the result of Boolean.parseBoolean(null) always false? Would be an indirect default ;)

Reply:
yeah, you're right, but a "false" default is more immediate to read :)
taking advantage of this opportunity to improve our setting resolver is good anyway

Reply:
Done - I added getBoolean to Config/SettingResolver but since Boolean.parseBoolean(..) will parse every String and null to false except for true (ignoreCase), I also added a value-validation before to only allow true and false as allowed settings values (with ignorecase). Otherwise I could have used the getNumber and just adjust the naming. - please check again if ok